src/AmbientSounds.Uwp/Views/ScreensaverPage.xaml.cs

```diff
@@ -232,5 +242,37 @@ private void OnToggleFullscreen(object sender, RoutedEventArgs e)
                 });
             }
         }
+
+        private void SetTimer()
+        {
+            InactiveTimer = new DispatcherTimer
```

Comment:
This API is deprecated, use `DispatcherQueueTimer` instead.

Reply:
> This API is deprecated

Sorry, I didn't find any reference, which says that it's deprecated. Can you provide a link with this information?

> use `DispatcherQueueTimer` instead

```DispatcherQueueTimer``` doesn't allow to work with UI elements.

Reply:
It's not "deprecated", it's like "soft deprecated". It just shouldn't be used at all anymore for new code. It also doesn't work at all on WinUI 3, which is another reason why it shouldn't be used for new code, as it'd make it harder to potentially migrate. `DispatcherQueue` and `DispatcherQueueTimer` should replace all `CoreDispatcher` and `DispatcherTimer` uses.

> "`DispatcherQueueTimer` doesn't allow to work with UI elements."

Not sure what you mean at all, and that doesn't sound correct. It's whole point is to do exactly this.

See: https://docs.microsoft.com/en-us/uwp/api/windows.system.dispatcherqueuetimer.

Reply:
> _current_ dispatcher queue

```Object reference not set to an instance of an object```

I'm not an expert of UWP, this is something new for me.